apps/frontend/src/layout/AppFooter.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { Typography, Button, Row, Col } from "antd";
+import { MailOutlined } from "@ant-design/icons";
+import { StyledFooter, StyledSubscribeInput, StyledCopyrightRow } from "./AppFooter.style";
+const { Title, Text, Link } = Typography;
+
+
+
+const AppFooter = () => {
+    return (
+        <StyledFooter>
+            <Row gutter={[32, 32]}>
+                {/* Column 1: Subscribe */}
+                <Col xs={24} sm={12} md={6}>
+                    <Title level={4}>
+                        Exclusive
+                    </Title>
+                    <Text>Subscribe</Text>
+                    <Text>
+                        Get 10% off your first order
+                    </Text>
+                    <StyledSubscribeInput
+                        placeholder="Enter your email"
+                        suffix={<MailOutlined />}
+                    />
+                    <Button>
+                        Subscribe
+                    </Button>
+                </Col>
+
+                {/* Column 2: Support */}
+                <Col xs={24} sm={12} md={6}>
+                    <Title level={5} >
+                        Support
+                    </Title>
+                    <Text>
+                        111 Bijoy Sarani, Dhaka, DH 1515, Bangladesh.
+                    </Text>
+                    <Text>exclusive@gmail.com</Text>
+                    <Text>+88015-88888-9999</Text>
+                </Col>
+
+                {/* Column 3: Account */}
+                <Col xs={24} sm={12} md={6}>
+                    <Title level={5}>
+                        Account
+                    </Title>
+                    <ul>
+                        <li>
```

Comment:
Having a list of titles and render them within a for-loop is better to maintain.